microsoft-365/security/office-365-security/remediate-malicious-email-delivered-office-365.md

```diff
@@ -125,8 +125,13 @@ Open any remediation item to view details about it, including its name, creation
 
   - **Failure**: The desired action on remediable emails failed. For example: An admin wants to remove emails from mailboxes, so the admin takes the action of soft-deleting emails. If a remediable email is still found in the mailbox after the action is taken, status will show as failed.
 
+    **NEW**: Addition of column "Already in destination" in Action Log: Since we got the latest delivery location in Explorer and we want to utilize it for making smarter moves across the experiences, now, we are utilizing the latest delivery location in remediation by not sending signals to remediate if the message has already been remediated. This will help you understand the exact number of messages which need to be remediated. As we will not be remediating already remediated messages with same actions, we are introducing another column named "Already in destination". This means that the action you wish to perform was taken previously on the email, hence the email is already present in the destination folder. Since actions can only be taken on messages in inbox, junk, deleted folder and soft deleted folder from Explorer, these are remediable messages. So this column shows only for messages which are remediable. For instance, soft delete action is performed on the message present in inbox, then the message will get successfully actioned. Next time, when soft delete is performed on the same message again, it will show under the column "Already in destination".
```

Comment:
```suggestion
  - **NEW**: Addition of column "Already in destination" in Action Log: Since we've got the latest delivery location in Explorer and we want to utilize it for making smarter moves across the experiences, we are utilizing the latest delivery location in remediation by not sending signals to remediate if the message has already been remediated. This will help you understand the exact number of messages that need to be remediated. As we will not be remediating already remediated messages with the same actions, we are introducing another column named "Already in destination". This means that the action you wish to perform was taken previously on the email, hence the email is already present in the destination folder. Since actions can only be taken on messages in inbox, junk, deleted folder, and soft deleted folder from Explorer, these are remediable messages. So this column shows only messages that are remediable. For instance, a soft delete action is performed on the message present in the inbox, then the message will get successfully actioned. The next time a soft delete is performed on the same message, it will show under the column "Already in destination".
```